habitat_baselines/agents/benchmark_gym.py

```diff
@@ -279,9 +278,9 @@ def act(self, observations):
             #  Make masks not done till reset (end of episode) will be called
             self.not_done_masks.fill_(True)
             self.prev_actions.copy_(actions)  # type: ignore
-        action = action_array_to_dict(self.action_space, actions[0])
-
-        return action["action"]
+        # action = action_array_to_dict(self.action_space, actions[0])
+        # return action["action"]
```

Comment:
Need advice here. What is RearrangePPOAgent and how do I test it?

Reply:
This code is depreciated and should be removed actually. I originally had it to evaluate the motion planning and learned approaches in the same script. However, the code is not maintained.